qsirecon/config.py

```diff
@@ -553,6 +553,20 @@ class workflow(_Config):
     """Recon workflow specification."""
     output_resolution = None
     """Isotropic voxel size for outputs."""
+    qsirecon_suffixes = []
```

Comment:
I added the list of qsirecon_suffixes to the Config object. I figured it could be useful since we reference them in a number of places.